tests/test_service/test_adapter.py

```diff
@@ -270,10 +270,14 @@ def move_tif(*args, **kwargs):
             mock_reproject.call_args_list, expected_reproject_calls
         ):
             np.testing.assert_array_equal(
-                actual_call.kwargs['source'], expected_call.kwargs['source']
+                actual_call.kwargs['source'],
+                expected_call.kwargs['source'],
+                strict=True,
```

Comment:
adding `strict=True` forces the comparison to "[raise an AssertionError when either the shape or the *data type* of the array_like objects does not match](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html)."